### ISSUE TITLE
make('figures') throws error on Windows

### ISSUE THREAD

**Issue by johnbaums** [traitecoevo/remake_baad_example#1]
On Windows (session info at bottom), `remake::make("figures")` gives:

```
[  LOAD ] 
[  READ ]                                 |  # loading sources
<  MAKE > figures
[ DLOAD ] downloads/BTFF15.jpg            |  downloader::download("https://www.wow247.co.uk/wp-content/uploads/2015/09/back-to-the-future-poste...
[  READ ]                                 |  # loading packages
trying URL 'https://www.wow247.co.uk/wp-content/uploads/2015/09/back-to-the-future-poster.jpg'
Content type 'image/jpeg' length 106610 bytes (104 KB)
downloaded 104 KB

[ DLOAD ] downloads/michael.png           |  downloader::download("https://raw.githubusercontent.com/dfalster/baad/master/extra/baad.png", "dow...
trying URL 'https://raw.githubusercontent.com/dfalster/baad/master/extra/baad.png'
Content type 'image/png' length 83322 bytes (81 KB)
downloaded 81 KB

[ DLOAD ] downloads/baad.rds              |  downloader::download("https://github.com/dfalster/baad/releases/download/v1.0.0/baad.rds", "downlo...
trying URL 'https://github.com/dfalster/baad/releases/download/v1.0.0/baad.rds'
Content type 'application/octet-stream' length 1217875 bytes (1.2 MB)
downloaded 1.2 MB

[ BUILD ] baad                            |  baad <- readRDS("downloads/baad.rds")
Error in readRDS("downloads/baad.rds") : 
  embedded nul in string: 'Fatemi\004\0\t\0'
```

BTFF15.jpg is then in the downloads folder, and looks like this:

![btff15](https://cloud.githubusercontent.com/assets/818438/11618808/af6fd6e2-9cf3-11e5-9e2f-6374ffbdce65.jpg)

The only other image in the folder, michael.png, has not been closed so can't be viewed (though `dev.off()` suggests the device _has_ been closed: `Error in dev.off() : cannot shut down device 1 (the null device)`).

```
> sessionInfo()
R version 3.2.2 (2015-08-14)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 8 x64 (build 9200)

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252    LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] mapdata_2.2-5 maps_2.3-10   smatr_3.4-6   plyr_1.8.3    remake_0.2.0 

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.2      codetools_0.2-14 digest_0.6.8     crayon_1.3.1     rappdirs_0.3     R6_2.1.1         storr_0.2.0      magrittr_1.5    
 [9] httr_1.0.0       stringi_1.0-1    curl_0.9.4       devtools_1.9.1   tools_3.2.2      downloader_0.4   stringr_1.0.0    yaml_2.1.13     
[17] memoise_0.2.1   
```

**Comment by dfalster** [comment]
Thanks John. Urrgh.  Looks like download functionality is really bombing out. Could you try with latest commit (be54355db3200f762cb019aba39e48e806604767), to see if dataset part works (i'll remove the pictures later).

**Comment by johnbaums** [comment]
@dfalster - the pdf figures look fine now, but the .png and .jpg are still screwy.

**Comment by dfalster** [comment]
fixed in be54355db3200f762cb019aba39e48e806604767 and 43d79b3ea75c8a817f4fccf326cceb9b99570670
